### PR TITLE
Move to pip-tools off of Pipenv

### LINKED ISSUE #127: Move off of Pipenv to pip-tools
Pipenv's future is murky at best. pip-tools is very usable and good and nice.

### PR DESCRIPTION
**Pull request by wlonk**
Fixes #127

angy blepping kitteh

### CODE REVIEW

File: requirements/base.in [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+gunicorn~=19.9
+psycopg2-binary~=2.8
+pytz==2020.1
+whitenoise~=4.1
+django-crispy-forms~=1.8
+Django~=2.2
+django-registration~=3.0
+django-sendgrid-v5~=0.8
+djangorestframework~=3.10
+django-js-reverse~=0.9
```
Comment (ojacobson) [inline requirements/base.in]: I'm going to take on faith that these requirements are correct, as Github provides no straightforward way to review this kind of change.

File: README.rst [hunk 2]
```diff
@@ -29,10 +29,17 @@ root::
    export DJANGO_DEBUG=True
    export SENDGRID_API_KEY=INVALID
 
-Use ``pipenv`` to get the database into a good state::
+Set up your virtualenv with your preferred method. For me, using
+``virtualenvwrapper`` and ``pyenv``, this looks like::
 
-   $ pipenv sync --dev
-   $ pipenv run python manage.py migrate
```
Comment (ojacobson) [inline README.rst:35]: I could not be happier to see `pipenv run` gone.

File: README.rst [hunk 3]
```diff
@@ -37,6 +37,10 @@ Set up your virtualenv with your preferred method. For me, using
    $ pip install pip-tools
    $ pip-sync requirements/dev.txt requirements/base.txt
 
+You'll have to read the contents of your ``.env`` file into your local
+shell's environment. You may want to set up a system to do this each
+time you activate this virtualenv.
+
```
Comment (ojacobson) [inline README.rst:43]: Links to prior art would help, here. "You'll need to do this thing" on its own is kinda contributor-hostile, even if it is inadvertent.

File: requirements/base.in [hunk 4]
```diff
@@ -4,7 +4,7 @@ psycopg2-binary~=2.8
 pytz==2020.1
 whitenoise~=4.1
 django-crispy-forms~=1.8
-Django~=2.2
+Django~=3.0
```
Comment (ojacobson) [inline requirements/base.in]: This feels like a major change to mix into this! Should it be a separate PR?

File: requirements/base.in (hunk 1 above)
Comment (wlonk) [reply requirements/base.in]: They're the same as it was, just changed to `~=`s.

File: README.rst (hunk 3 above)
Comment (wlonk) [reply README.rst:43]: True. I'll ponder how to write that up.

File: requirements/base.in (hunk 4 above)
Comment (wlonk) [reply requirements/base.in]: As you commented on #138, Django 3 is a surprisingly small update. I am content to have this in here, albeit in a distinct commit, for that reason.

File: README.rst (hunk 3 above)
Comment (ojacobson) [reply README.rst:43]: Don't hold this PR up on it, unless there's a clear answer.
